app/controllers/users/sessions_controller.rb

```diff
@@ -91,13 +93,21 @@ def cache_active_profile
     end
 
     def user_signed_in_and_not_locked_out?(user)
```

Comment:
do we need the user to get passed in here? We should have access to `current_user` within this method, right?